app/scripts/controllers/createClientBuild.js

```diff
@@ -131,6 +131,15 @@ angular.module('openshiftConsole')
         };
       }
 
+      if (clientConfig.envVars) {
+        _.forEach(clientConfig.envVars, function(envVar) {
+          if (envVar.name) {
+            var env = _.omit(envVar, ['$$hashKey']);
```

Comment:
We may remove this is [this property](https://github.com/openshift/origin-web-console/blob/5de5ca414129549aab483c7e9ab08a87cbf0e4fa/app/views/directives/edit-environment-variables.html#L17) removes the `$$hash` property

Reply:
Ok. I think `angular.toJson` is probably best.

```.js
var envVars = _.map(clientConfig.envVars, angular.toJson);
```
?

Reply:
sure, will change it to that. :+1: